requirements-python.txt

```diff
@@ -1,5 +1,5 @@
 PyYAML==5.1
-libvirt-python==4.9.0
+libvirt-python==5.7.0
```

Comment:
@bentsi is that so ? I'll be more then happy to remove them

Reply:
Added one commit to remove libvirt and openstack

Reply:
I don't mind to remove the support of open stack and libvirtv since it is completely broken, but it should be part of another PR (this is not related to porting SCT to python 3) and we need approval from @roydahan